api/V1/File.js

```diff
@@ -132,10 +133,15 @@ module.exports = (app, baseUrl) => {
         key: file.fileKey
       };
 
+      const s3Data = await util.getS3Object(file.fileKey).catch(err => {
+        return responseUtil.serverError(response, err);
+      });
+
       return responseUtil.send(response, {
         statusCode: 200,
         messages: [],
         file: file,
+        fileBytes: s3Data.ContentLength,
```

Comment:
I wonder if `fileSize` would be a better name? `fileBytes` makes me think of the contents of the file.